apis/r/R/utils-arrow.R

```diff
@@ -112,18 +112,35 @@ arrow_type_range <- function(x) {
     uint32 = bit64::as.integer64(c(0, 4294967295)),
     # We can't specify the full range of uint64 in R so we use the max of int64
     uint64 = c(bit64::as.integer64(0), bit64::lim.integer64()[2]),
-    float32 = c(-3.4028235e+38, 3.4028235e+38),
+    # float32/float
     float = c(-3.4028235e+38, 3.4028235e+38),
-    float64 = c(-1.7976931348623157e+308, 1.7976931348623157e+308),
+    # float64/double
     double =  c(-1.7976931348623157e+308, 1.7976931348623157e+308),
```

Comment:
If you want to be fancy you could use R constants here:

```r
> c(.Machine$double.xmin, .Machine$double.xmax)
[1] 2.22507e-308 1.79769e+308
> 
```

Reply:
I _always_ want to be fancy 😄. Great suggestion!

Reply:
Fixed in d20770aa and tested in 45b1a7ad